src/core_plugins/getting_started/public/components/injected_items/injected_items.js

```diff
@@ -6,7 +6,7 @@ import template from './injected_items.html';
 import './injected_items.less';
 
 function makeAngularParseableExpression(item) {
-  return `<div>${item}&nbsp;</div>`;
+  return `<div class="injected-items-item">${item}&nbsp;</div>`;
```

Comment:
I think per our [CSS styleguide](https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md) this should be named something like `injectedItems__item`. @cjcenizal?